### PR TITLE
Updating current switches

### PR DESCRIPTION
**Pull request by sofietoft**
**This PR is dependant on
This Docs PR: https://github.com/umbraco/UmbracoDocs/pull/3092
This Our PR: https://github.com/umbraco/OurUmbraco/pull/693**

Also added a new one for the new Getting Started section.

### CODE REVIEW

**Review by sigrundis — APPROVED** [review]
It is a bit challenging to test this because the new Getting Started section is missing and the title of the section that will be Fundamentals is still set as Getting Started. So basically all links are broken on this branch but technically it is how it is supposed to be 😅 

I don't see any obvious mistakes, as long as I am understanding the task correctly:

-All links that are in the Getting Started section are now directed to documentation/Fundamentals/* in stead of /documentation/Getting-Started/*
-Added links for the new Getting Started Section.

What I suggest we do in order to test this properly before deploying is to change this PR from merging to main to merging to the [New Getting Started PR](https://github.com/umbraco/UmbracoDocs/pull/3092) PR along with [PR 693](https://github.com/umbraco/OurUmbraco/pull/693). 

Then we can test everything together on the newGettingStarted branche before merging that one to main 😊

**Comment by sofietoft** [comment]
OK. We can also "just" merge this in after the PR on the docs has been merged, if that's better?

That way you should also be able to test this PR against the latest docs.
I think that the most important thing to coordinate here was [PR 693](https://github.com/umbraco/OurUmbraco/pull/693), as that should ensure no broken links once the Docs PR is merged in.

So, we could merge the Docs PR and the rewrites PR now(ish), and then test out the menu structure afterwards?
The menu structure will still be built on Our as far as I remember, it will just put the links into alphabetic order, if the switch isn't updated!

**Comment by sigrundis** [comment]
Yes, that also works 😊 
Just let me know when this is ready for testing!

**Pull request merged**
